test/e2e/framework/node/resource.go

```diff
@@ -400,16 +401,22 @@ func GetMasterAndWorkerNodes(c clientset.Interface) (sets.String, *v1.NodeList,
 	for _, n := range all.Items {
 		if system.IsMasterNode(n.Name) {
 			masters.Insert(n.Name)
-		} else if isNodeSchedulable(&n) && isNodeUntainted(&n) {
+		} else if IsNodeSchedulable(&n) && IsNodeUntainted(&n) {
 			nodes.Items = append(nodes.Items, n)
 		}
 	}
 	return masters, nodes, nil
 }
 
-// Test whether a fake pod can be scheduled on "node", given its current taints.
+// IsNodeUntainted tests whether a fake pod can be scheduled on "node", given its current taints.
 // TODO: need to discuss wether to return bool and error type
-func isNodeUntainted(node *v1.Node) bool {
+func IsNodeUntainted(node *v1.Node) bool {
+	return IsNodeUntaintedWhitelist(node, nil)
+}
+
+// IsNodeUntaintedWhitelist tests whether a fake pod can be scheduled on "node"
+// but allows for taints matching the regexp of whitelisted values.
+func IsNodeUntaintedWhitelist(node *v1.Node, ignoreTaints *regexp.Regexp) bool {
```

Comment:
IsNodeUntaintedWhitelist() is called in package node only, so it is not necessary to expose it.

Reply:
Done.